README.md

```diff
@@ -9,6 +9,52 @@
 
 `composer require lullabot/mpx-php`
 
+## Example
+
+```php
+<?php
+
+use Cache\Adapter\PHPArray\ArrayCachePool;
+use Lullabot\Mpx\Client;
+use Lullabot\Mpx\DataService\DataObjectFactory;
+use Lullabot\Mpx\DataService\DataServiceManager;
+use Lullabot\Mpx\Service\IdentityManagement\UserSession;
+use Lullabot\Mpx\TokenCachePool;
+use Lullabot\Mpx\User;
+use Psr\Log\NullLogger;
+
+// Only required if your application is not using Composer's autoloader already.
+require_once './vendor/autoload.php';
+
+// Create a new MPX client with the default configuration.
+$defaults = Client::getDefaultConfiguration();
+$client = new Client(new GuzzleHttp\Client($defaults));
+
+// Replace your username and password here. The username must begin with `mpx/`.
+$user = new User('mpx/YOU@EXAMPLE.COM', 'secret');
+$tokenCachePool = new TokenCachePool(new ArrayCachePool());
+$session = new UserSession($client, $user, $tokenCachePool, new NullLogger());
+
+// This registers the annotation loader.
+$dataServiceManager = DataServiceManager::basicDiscovery();
+
```

Comment:
Looking at all that above, from a developer experience perspective, I think we could simplify all those lines a lot. In this example, the data input is just for the user ('mpx/YOU@EXAMPLE.COM' and 'secret'). All the rest of lines are trivial initialization objects that we could save to developers if we implemented a class/method that receives the username and password and set all the rest of default stuff internally, giving us the session, dataServiceManager, and/or whatever the developer might need to continue.

Reply:
I added an issue for just this suggestion over at #69.

Reply:
Great! :+1: